app/src/main/java/com/example/mycleaninglog/MainViewModel.kt

```diff
@@ -118,6 +118,11 @@ class MainViewModel : ViewModel() {
         handle.addOnFailureListener{Log.e("Firebase", "Save Failed $it")}
     }
 
+    fun deleteCleaningTask(preConTask: cleaningTask, selectedRoom: myRoom){
+        val document = firestore.collection("users").document(user.uid).collection("cleaningTasks").document(preConRoom.uniqueID)
+        document.delete()
+    }
+
```

Comment:
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.